app/Http/Middleware/SubscribersConfigured.php

```diff
@@ -52,10 +52,6 @@ public function __construct(Repository $config)
      */
     public function handle(Request $request, Closure $next)
     {
-        if (!$this->config->get('setting.enable_subscribers')) {
```

Comment:
Why have we removed this condition?

Reply:
> Why have we removed this condition?

The subscription page must be manageable, despite the setting "Allow to subscribe". Admin should always be able to configure the subscribers and manage their subscriptions, etc. 

You can not have this setting enabled or not based on "if the user is allowed to subscribe or not feature"

Reply:
Then the condition should check for an active admin user, then?

Reply:
It is done at the auth module ( outside of the scope for this commit)

Reply:
Why can't we add a condition into https://github.com/CachetHQ/Cachet/blob/2.4/app/Http/Middleware/SubscribersConfigured.php for admin?

Reply:
It is handled in app/Http/Kernel.php 
@jbrooksuk

Reply:
I don't understand?

Reply:
If subscriptions aren't enabled and you're **not** an admin, then you shouldn't be able to see the subscription page, so as far as I can tell, the middleware is required.